File: app/src/main/java/com/like/example/ListActivity.java
package com.like.example;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.RecyclerView;

import com.like.example.adapters.ListAdapter;

import butterknife.BindView;
import butterknife.ButterKnife;

public class ListActivity extends AppCompatActivity {

    @BindView(R.id.list_view)
    RecyclerView listView;
    private ListAdapter adapter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_list);
        ButterKnife.bind(this);

        adapter = new ListAdapter(this);
        listView.setAdapter(adapter);

    }
}

File: app/src/main/java/com/like/example/ListActivity.kt
package com.like.example

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.RecyclerView

import com.like.example.adapters.ListAdapter

import butterknife.BindView
import butterknife.ButterKnife

class ListActivity : AppCompatActivity() {

    @BindView(R.id.list_view)
    lateinit var listView: RecyclerView
    private var adapter: ListAdapter? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list)
        ButterKnife.bind(this)

        adapter = ListAdapter(this)
        listView!!.adapter = adapter

    }
}
